Traceback (most recent call last):
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/client.py", line 1085, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/client.py", line 551, in async_execute
    await self.async_execute_cell(
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/client.py", line 846, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/uteamerstorfer/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/nbclient/client.py", line 748, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
model_svm = svm.SVC(gamma='auto') # specify the model/algorithm
model_svm.fit(X_train, y_train) # train the model with the training data
y_pred_svm = model_svm.predict(X_test) # make the prediction with the test data

print('%s: %f' %('accuracy (SVM)', metrics.accuracy_score(y_test, y_pred_svm)))

kfold = StratifiedKFold(n_splits=5, random_state=1, shuffle=False)
cv_score_svm = cross_val_score(model_svm, X_train, y_train, scoring='accuracy', cv=kfold)

print('%s: %f (%f)' %('accuracy cross-validation (standard deviation)', cv_score_svm.mean(), cv_score_svm.std()))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/var/folders/x3/2bzh843n0tv469w6l6sd8sq00000gn/T/ipykernel_65845/1998379179.py[0m in [0;36m<module>[0;34m[0m
[1;32m      5[0m [0mprint[0m[0;34m([0m[0;34m'%s: %f'[0m [0;34m%[0m[0;34m([0m[0;34m'accuracy (SVM)'[0m[0;34m,[0m [0mmetrics[0m[0;34m.[0m[0maccuracy_score[0m[0;34m([0m[0my_test[0m[0;34m,[0m [0my_pred_svm[0m[0;34m)[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      6[0m [0;34m[0m[0m
[0;32m----> 7[0;31m [0mkfold[0m [0;34m=[0m [0mStratifiedKFold[0m[0;34m([0m[0mn_splits[0m[0;34m=[0m[0;36m5[0m[0;34m,[0m [0mrandom_state[0m[0;34m=[0m[0;36m1[0m[0;34m,[0m [0mshuffle[0m[0;34m=[0m[0;32mFalse[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      8[0m [0mcv_score_svm[0m [0;34m=[0m [0mcross_val_score[0m[0;34m([0m[0mmodel_svm[0m[0;34m,[0m [0mX_train[0m[0;34m,[0m [0my_train[0m[0;34m,[0m [0mscoring[0m[0;34m=[0m[0;34m'accuracy'[0m[0;34m,[0m [0mcv[0m[0;34m=[0m[0mkfold[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      9[0m [0;34m[0m[0m

[0;32m~/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/sklearn/model_selection/_split.py[0m in [0;36m__init__[0;34m(self, n_splits, shuffle, random_state)[0m
[1;32m    646[0m [0;34m[0m[0m
[1;32m    647[0m     [0;32mdef[0m [0m__init__[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mn_splits[0m[0;34m=[0m[0;36m5[0m[0;34m,[0m [0;34m*[0m[0;34m,[0m [0mshuffle[0m[0;34m=[0m[0;32mFalse[0m[0;34m,[0m [0mrandom_state[0m[0;34m=[0m[0;32mNone[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 648[0;31m         [0msuper[0m[0;34m([0m[0;34m)[0m[0;34m.[0m[0m__init__[0m[0;34m([0m[0mn_splits[0m[0;34m=[0m[0mn_splits[0m[0;34m,[0m [0mshuffle[0m[0;34m=[0m[0mshuffle[0m[0;34m,[0m [0mrandom_state[0m[0;34m=[0m[0mrandom_state[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    649[0m [0;34m[0m[0m
[1;32m    650[0m     [0;32mdef[0m [0m_make_test_folds[0m[0;34m([0m[0mself[0m[0;34m,[0m [0mX[0m[0;34m,[0m [0my[0m[0;34m=[0m[0;32mNone[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/miniconda3/envs/epn-ml-book/lib/python3.9/site-packages/sklearn/model_selection/_split.py[0m in [0;36m__init__[0;34m(self, n_splits, shuffle, random_state)[0m
[1;32m    294[0m [0;34m[0m[0m
[1;32m    295[0m         [0;32mif[0m [0;32mnot[0m [0mshuffle[0m [0;32mand[0m [0mrandom_state[0m [0;32mis[0m [0;32mnot[0m [0;32mNone[0m[0;34m:[0m  [0;31m# None is the default[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 296[0;31m             raise ValueError(
[0m[1;32m    297[0m                 [0;34m"Setting a random_state has no effect since shuffle is "[0m[0;34m[0m[0;34m[0m[0m
[1;32m    298[0m                 [0;34m"False. You should leave "[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: Setting a random_state has no effect since shuffle is False. You should leave random_state to its default (None), or set shuffle=True.
ValueError: Setting a random_state has no effect since shuffle is False. You should leave random_state to its default (None), or set shuffle=True.

